### ISSUE TITLE
Add Tests, Tools and Guidelines for Content Accessibility

### ISSUE THREAD

**Issue by rafaelcastrocouto** [godotengine/godot-proposals#6489]
### Describe the project you are working on

https://github.com/rafaelcastrocouto/shotcaller-godot
Minimalistic 1v1 MOBA/RTS in top-down pixel style

### Describe the problem or limitation you are having in your project

Having a hard time assuring that I'm providing accessible interfaces.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

The current User interface has very limited features that help developers to deploy accessible content.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Content Accessibility Tests, Tools and Guidelines should provide means for making games, applications, and other content accessible to people with disabilities. 

- Tests would allow us to check if the editor itself and developed games are not breaking simple and testable accessibility rules.

- Tools would make easier by enable and validate accessible environments and providing quick shortcuts to set them up. 

- Guidelines are not required but provide insights in order to develop more accessible games.

Here's a very useful list of guidelines that can also be used to generate tools and tests https://www.w3.org/WAI/WCAG21/Techniques/

### If this enhancement will not be used often, can it be worked around with a few lines of script?

N/A

### Is there a reason why this should be core and not an add-on in the asset library?

N/A
